lib/galaxy/tools/deps/mulled/mulled_build_channel.py

```diff
@@ -19,7 +19,6 @@
 from __future__ import print_function
 
 import os
-import subprocess
```

Comment:
Bad merge, all changes to this file needs to be reverted, see commits c85db3fd68590d3d2f61b0b3f88026773d9563a0,19d03edc7ff8384a5360d71365076a2594fb0b8b and 626a3dc2995f0d797232aa666f1bdf3e6a5d22dd.

Reply:
Yeah - I really messed this up - I keep saying I need to merge these two projects much more often and then it slips as a priority. Thanks for keeping your eagle eyes on this - I'll try to do better.